packages/ifpack2/src/Ifpack2_OverlappingRowMatrix_def.hpp

```diff
@@ -840,6 +845,19 @@ OverlappingRowMatrix<MatrixType>::getUnderlyingMatrix() const
   return A_;
 }
 
+template<class MatrixType>
+Teuchos::RCP<const Tpetra::RowMatrix<typename MatrixType::scalar_type, typename MatrixType::local_ordinal_type, typename MatrixType::global_ordinal_type, typename MatrixType::node_type> >
+OverlappingRowMatrix<MatrixType>::getExtMatrix() const
+{
+  return ExtMatrix_;
```

Comment:
If `ExtMatrix_` always exists, you could return a const reference instead of an RCP.